app/components/banner/Banner.tsx

```diff
@@ -7,17 +6,18 @@ import { HeaderBanner } from 'cloudify-ui-components';
 import i18n from 'i18next';
 import Consts from '../../utils/consts';
 import LicenseLabel from '../LicenseLabel';
+import type { ReduxState } from '../../reducers';
 
 interface BannerProps {
     className?: string;
 }
 
 function Banner({ className = '' }: BannerProps) {
-    const isCommunity = useSelector(
-        state => _.get(state, 'manager.version.edition', Consts.EDITION.PREMIUM) === Consts.EDITION.COMMUNITY
-    );
-    const licenseEdition = useSelector(state => _.get(state, 'manager.license.data.license_edition', ''));
-    const productVersion = useSelector(state => _.get(state, 'manager.version.version', ''));
+    const isCommunity =
+        useSelector((state: ReduxState) => state.manager.version.edition || Consts.EDITION.PREMIUM) ===
+        Consts.EDITION.COMMUNITY;
+    const licenseEdition = useSelector((state: ReduxState) => state.manager.license.data?.license_edition || '');
+    const productVersion = useSelector((state: ReduxState) => state.manager.version.version || '');
```

Comment:
To be in line with the original implementation I think it should be rather like this:
```suggestion
    const isCommunity =
        useSelector((state: ReduxState) => state.manager.version.edition ?? Consts.EDITION.PREMIUM) ===
        Consts.EDITION.COMMUNITY;
    const licenseEdition = useSelector((state: ReduxState) => state.manager.license.data?.license_edition ?? '');
    const productVersion = useSelector((state: ReduxState) => state.manager.version.version ?? '');
```

Reply:
I changed: [322bcff](https://github.com/cloudify-cosmo/cloudify-stage/pull/2247/commits/322bcff026966dd1e8f81cc79379e758f6ba7e3d)